core/frontend/src/tile/TileAdmin.ts

```diff
@@ -30,6 +30,7 @@ export interface ExternalTileStatistics {
   requested: number;
   selected: number;
   ready: number;
+  hasMissingData?: number;
```

Comment:
This name doesn't make sense for a numeric property.
Did you consider simply having OrbitGT tree add 1 to `requested` if it has missing data?

Reply:
Yes that was my first thought, but it felt a bit obscure to sort of lie about the meaning of `requested`.  Do you think that seems better?  We could go that route.

Reply:
reworked this to not expose hasMissingData - instead bump up number of requested by 1.